### PR TITLE
Update upload.sh for using pyprojet.toml and setup.cfg files

### PR DESCRIPTION
**Pull request by thehomebrewnerd**
Updates for using setup.cfg and pyproject.toml files:
- Get library name from `setup.cfg`
- Get version using updated command and library name
- Update build command to use `python -m build` to create sdist and wheel

### CODE REVIEW

**Review by gsheni — DISMISSED** [review]

File: upload.sh [hunk 1]
```diff
@@ -60,7 +63,10 @@ upload_package() {
 
     # Upgrade pip.
     python -m pip install --quiet --upgrade pip
-
+    
+    # Upgrade build.
+    python -m pip install --quiet --upgrade build
```
Comment (rwedge) [inline upload.sh]: isn't build getting used in `build_package`?  should we upgrade earlier?

Comment (thehomebrewnerd) [reply upload.sh]: Yeah, good catch. Should we create the venv and upgrade pip and build before we call `build_package`?

Comment (gsheni) [reply upload.sh]: Is there a reason we were not creating a venv before? Do we need to do so now?

Comment (thehomebrewnerd) [reply upload.sh]: We were creating it but only before the twine install, not for the build.

Comment (rwedge) [reply upload.sh]: unsure, perhaps the twine install made a venv necessary

Comment (rwedge) [reply upload.sh]: doing it all in a venv is fine with me

**Review by rwedge — APPROVED** [review]

**Pull request merged**